### ISSUE TITLE
Voor dat een nieuwe zet op het bord komt een geluidsfragment afspelen

### ISSUE THREAD

**Issue by Tsluiz** [bart314/boterkaaseieren#17]

**Comment by bytewalker** [comment]
Ja, pimpen met geluid en het liefst met grafische interface :-)

**Comment by ketelbinkie** [comment]
Taco, ik heb je code iets aangepast (zie pullrequest 'soundForEveryOne#17'), de verwijzing naar user/taco/.... etc werkt niet bij iedereen ;-). .wav file verplaatst naar een directory 'resources' en de verwijzing naar deze directory opgenomen. Hopelijk werkt het straks ook nog bij anderen ;-)
